### ISSUE TITLE
require 'busted.runner'() fails

### ISSUE THREAD

**Issue by romanchyla** [Olivine-Labs/busted#490]
As said, this will fail with the foloowing error:

```
lua tests/iotzfutils_spec.lua 
lua: ...al/share/lua/5.1/busted/modules/test_file_loader.lua:25: bad argument #1 to 'find' (string expected, got nil)
stack traceback:
	[C]: in function 'find'
	...al/share/lua/5.1/busted/modules/test_file_loader.lua:25: in function 'pred'
	/usr/local/share/lua/5.1/pl/tablex.lua:634: in function 'filter'
	...al/share/lua/5.1/busted/modules/test_file_loader.lua:24: in function 'getTestFiles'
	...al/share/lua/5.1/busted/modules/test_file_loader.lua:45: in function 'getAllTestFiles'
	...al/share/lua/5.1/busted/modules/test_file_loader.lua:60: in function 'testFileLoader'
	/usr/local/share/lua/5.1/busted/runner.lua:167: in function </usr/local/share/lua/5.1/busted/runner.lua:9>
	tests/iotzfutils_spec.lua:1: in main chunk
	[C]: ?
```

the error happens because the pattern is 'nil', if I specify patter in my .busted config, it works

(when i run the code using 'busted' it works fine without the require) - I think the getTestFiles() funciton should detect/deal with the pattern being 'nil'

my version is:  2.0.rc10-0 (installed) - /usr/local/lib/luarocks/rocks

**Comment by o-lim** [comment]
I couldn't reproduce the same error, but just in case, this should be fixed with PR#493.  You might want to try to see if this also happens with rc11.

**Comment by romanchyla** [comment]
Wow, thanks! I will try to test and get back if there is some new development.
